platform/core/src/classes/CommandsManager.ts

```diff
@@ -167,21 +167,31 @@ export class CommandsManager {
    * Run one or more commands with specified extra options.
    * Returns the result of the last command run.
    *
-   * @param toRun - A specification of one or more commands
+   * @param toRun - A specification of one or more commands,
+   *  typically an object of { commandName, commandOptions, context }
+   * or an array of such objects. It can also be a single commandName as string
+   * if no options are needed.
    * @param options - to include in the commands run beyond
    *   the commandOptions specified in the base.
    */
   public run(
-    toRun: Command | Commands | Command[] | undefined,
+    toRun: Command | Commands | Command[] | string | undefined,
```

Comment:
If Command was string | { commandName: string, ...} and you check here, then the existing case/typedef up top works, and you can still just call it as a straight string - eg
commands: "singleCOmmandName"
or
commands: [
  "singleCOmmandName",
  { commandName: "complexCommand", commandOptions: {a:1}},
]

That also make the array conversion just work with string already.